cli/command/swarm/progress/root_rotation.go

```diff
@@ -0,0 +1,171 @@
+package progress
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/progress"
+	"github.com/docker/docker/pkg/streamformatter"
+	digest "github.com/opencontainers/go-digest"
+)
+
+const (
+	certsRotatedStr = "  nodes rotated TLS certificates"
+	rootsRotatedStr = "   nodes rotated CA certificates"
```

Comment:
I'm sure there's a good reason, but why are there a different number of spaces before these?